spec/models/person_spec.rb

```diff
@@ -658,12 +658,12 @@
   describe '#as_json' do
     it 'returns a hash representation of a person' do
       expect(@person.as_json).to eq({
-        :id => @person.id,
-        :guid => @person.guid,
-        :name => @person.name,
-        :avatar => @person.profile.image_url(:thumb_medium),
-        :handle => @person.diaspora_handle,
-        :url => Rails.application.routes.url_helpers.person_path(@person),
+        id:     @person.id,
```

Comment:
Layout/IndentHash: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.